android/src/main/java/com/tailscale/ipn/MainActivity.kt

```diff
@@ -285,6 +293,34 @@ class MainActivity : ComponentActivity() {
     lifecycleScope.launch { Notifier.loginFinished.collect { _ -> loginQRCode.set(null) } }
   }
 
+  private fun showPermissionDeniedDialog() {
```

Comment:
Perhaps, use a function name more relevant to the purpose? For instance, "showOtherVPNConflictDialog()".